### ISSUE TITLE
Blank lines in View code

### ISSUE THREAD

**Issue by ctac2010** [dbeaver/dbeaver#4178]
I replace VIEW in Teradata with code:
```
REPLACE VIEW BASE.V_TEST 
AS
SELECT
    1       AS ID,
    'TEST'  AS NAME
```

When execute command **SHOW VIEW BASE. V_TEST**, get view text with blank lines:
```
REPLACE VIEW SNAPSHOT.V_TEST_KSS 

AS

SELECT

    1       AS ID,

    'TEST'  AS NAME
```

Why  empty lines in code are present?

**Comment by serge-rider** [comment]
I can't say (seems to be some Taradata-specific glitch) but it doesn't look like a DBeaver issue.

**Comment by serge-rider** [comment]
Do you see extra line feeds in other tools (for views created in DBeaver)?
This may happen if your view source contains "broken" line feeds like \n\r  (this bug will be fixed in the next version).

Are you typing view source manually or copy-paste it from some external program? If copy-paste then could you check what line feeds it has?

In any case - please check the fix in 5.2.1 or in EA (https://dbeaver.io/files/ea)
